doc/release-process.md

```diff
@@ -65,7 +67,9 @@ Release Process
 	mv build/out/bitcoin-*.tar.gz build/out/src/bitcoin-*.tar.gz ../
 	./bin/gbuild --commit bitcoin=v${VERSION} ../bitcoin/contrib/gitian-descriptors/gitian-win.yml
 	./bin/gsign --signer $SIGNER --release ${VERSION}-win --destination ../gitian.sigs/ ../bitcoin/contrib/gitian-descriptors/gitian-win.yml
```

Comment:
It could be, but that would require that everyone change their current build procedures. As it is, it's only an additional (optional) build step to attach a sig.

Changing it to match would be fine by me though if that's what's preferred.